projects/step-core/styles/_core-variables.scss

```diff
@@ -48,6 +48,7 @@ $yellow-15: #faebcc;
 $yellow-20: #ead5bd;
 $yellow-600: #ab793b;
 $yellow-650: #8a6d3b;
+$yellow-highlight: #fffdb0;
```

Comment:
According to color's definitions above, make sense to put it on top and rename as $yellow-5